packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -548,6 +662,22 @@ class ScrollableBaseNative extends Component {
 		}
 	}
 
+	initHorizontalOverscrollRef = (ref) => {
+		if (ref) {
+			this.overscrollRefs['horizontal'] = ReactDOM.findDOMNode(ref); // eslint-disable-line react/no-find-dom-node
```

Comment:
We could simply use `this.overscrollRefs.horizontal`.

Reply:
I will do it if the code exists after cleaning up. ;)